packages/main/src/plugin/image-registry.ts

```diff
@@ -134,8 +137,8 @@ export class ImageRegistry {
     if (exists) {
       throw new Error(`Registry ${registryCreateOptions.serverUrl} already exists`);
     }
-    const registry = provider.create(registryCreateOptions);
-    this.telemetryService.track('createRegistry', {
+    const registry = await provider.create(registryCreateOptions);
+    await this.telemetryService.track('createRegistry', {
```

Comment:
usually I was not using await to send events as I was not interested in waiting it's actually sent before returning the method

Reply:
removed await modifier